src/app/portlets/content-types/fields/content-type-fields-add-row/content-type-fields-add-row.component.scss

```diff
@@ -30,9 +30,13 @@ $row-height: 70px;
     align-items: baseline;
     height: 198px;
     justify-content: center;
-    text-align: center;
 
-    &.dot-add-rows__add {
+    ::ng-deep .ui-button.ui-button-icon-only {
```

Comment:
`_splitbutton.scss`

Reply:
so in the future no icons will be supported for all splitbuttons in the submenu